pkg/koordlet/statesinformer/states_device_linux.go

```diff
@@ -145,7 +145,7 @@ func (s *statesInformer) buildGPUDevice() []schedulingv1alpha1.DeviceInfo {
 		s.gpuMutex.RUnlock()
 		deviceInfos = append(deviceInfos, schedulingv1alpha1.DeviceInfo{
 			UUID:   gpu.DeviceUUID,
-			Minor:  gpu.Minor,
+			Minor:  &gpu.Minor,
```

Comment:
do not use the pointer of a for-range value

Reply:
everyday you learn something new